framework/src/transfers/MultiAppInterpolationTransfer.C

```diff
@@ -16,7 +16,7 @@
 #include "MooseTypes.h"
 #include "MooseVariableFE.h"
 #include "MultiApp.h"
-#include "MooseCoordTransform.h"
+#include "MooseAppCoordTransform.h"
 
 #include "libmesh/parallel_algebra.h"
```

Comment:
should you add the skipCoordTransform parameters everywhere?
  MultiAppTransfer::addSkipCoordCollapsingParam(params);

Reply:
I modeled this design after the bounding box expansion factor, which you were the conceptual pioneer of. I personally like only adding the parameter to classes for which both of the following are true:

1. The class supports the coordinate transformation object
2. Coordinate collapsing is potentially "problematic", e.g. `mapBack` is called, and/or users have leveraged a lack of coordinate collapsing in the past when desiging their inputs

I think that minimizing the use of `skip_coordinate_collapsing` is a good thing ... or else coordinate collapsing within the transformation object is just a bad feature if we find ourselves trying to avoid it a lot. And maybe it is 🤷 I've wondered that to myself.

Reply:
kk works for me.
transfers are very often interchangeable (as long as you dont care about projection errors), so if 2 applied to 2 transfers, it likely could apply to all of them